rules/group.go

```diff
@@ -265,6 +266,10 @@ func (g *Group) run(ctx context.Context) {
 			if missed > 0 {
 				g.metrics.IterationsMissed.WithLabelValues(GroupKey(g.file, g.name)).Add(float64(missed))
 				g.metrics.IterationsScheduled.WithLabelValues(GroupKey(g.file, g.name)).Add(float64(missed))
+				numRulesInGroup := uint64(len(g.rules))
```

Comment:
Seems we have two casts to `uint64` then a further one to `float64`.  
Can we skip any of them?